animated-gif/src/main/jni/gifimage/gif.cpp

```diff
@@ -1147,6 +1147,39 @@ jint GifFrame_nativeGetDurationMs(JNIEnv* pEnv, jobject thiz) {
   return spNativeContext->durationMs;
 }
 
+/**
+ * Gets the color (as an int, as in Android) of the transparent pixel of this frame
+ *
+ * @return the color (as an int, as in Android) of the transparent pixel of this frame
+ */
+jint GifFrame_nativeGetTransparentPixel(JNIEnv* pEnv, jobject thiz) {
+  auto spNativeContext = getGifFrameNativeContext(pEnv, thiz);
+
+  ColorMapObject* pColorMap = spNativeContext->spGifWrapper->get()->SColorMap;
```

Comment:
Hi, I am in the context of a single frame, not an image, so I am unclear what value to use as `frameNum` in the code fragment you provided.  Can you elaborate on how to get the correct color table from within the context of a frame (I don't know which frame number I am).